### PR TITLE
TET-98 Display node indexes (tetrahedron vertices)

### PR DESCRIPTION
**Pull request by Max1mj**
https://github.com/user-attachments/assets/dd98a379-777f-41cd-8c2a-3818a5f2aa1f
# Summary:
- Added `NodeDisplay` component
- Added `positions` display on `index` hover
- Removed `Experience` component because our entry point is `ModelViewPage`
- Removed `ambientLight` in `Scene` component because it didn't affect on figure display
- Removed `auto-added` files in previous commit

### CODE REVIEW

File: .vs/3d-app-web/v17/DocumentLayout.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (HappyMary16) [inline .vs/3d-app-web/v17/DocumentLayout.json:1]: looks like an autogenerated file. Do we really need it in our repo?

**Review by HappyMary16 — CHANGES_REQUESTED** [review]

**Comment by HappyMary16** [comment]
It's great that indexes are displayed but it doesn't match our design. You can talk to Tetiana to update a design or update implementation

**Comment by HappyMary16** [comment]
I would really like to see some progress in this PR

File: src/components/NodeDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { Html } from '@react-three/drei'
+import { FC, useMemo } from 'react'
+import * as THREE from 'three'
+
+interface NodeType {
+  positions: Float32Array
+}
+
+const NodeDisplay: FC<NodeType> = ({ positions }) => {
+  const indexedPositions = useMemo(() => {
```
Comment (HappyMary16) [inline src/components/NodeDisplay.tsx:10]: is it possible to check in unit tests that this indexedPositions has a correct value after element initialization?

**Review by HappyMary16 — APPROVED** [review]

**Review by ut5tw — APPROVED** [review]

**Pull request merged**